docs/run-testing-framework.md

```diff
@@ -86,10 +86,46 @@ cd terraform/eks && terraform init && terraform apply \
     -var="testcase=../testcases/{{your test case folder name}}" \
     -var-file="../testcases/{{your test case folder name}}/parameters.tfvars"
 ````
- 
+
+### 2.3.1 Run in EKS Fargate
+#### Set Up
+* Install kubectl and eksctl
```

Comment:
These should be provided as a terraform config for making it easier to run. If the reason to not include the k8s setup in the test tf files is the time it takes to provision a cluster (it is indeed best practice to decouple cluster creation and use) it's still good to provide these as a separate terraform deployment instead of manual steps.

Reply:
Sorry read your doc now and realized that this is your plan

Reply:
Yeah but here we're documenting complicated steps for starting up a cluster - instead we should provide a simple way to do it, with just a single terraform command. And it should make it easier in the future to move toward automatically bringing up the cluster in tests too. I don't think it's worth adding these complicated steps now instead of modeling them with terraform (a separate deployment, not part of the test, for now)